src/store/api/ApplicationApi.js

```diff
@@ -0,0 +1,57 @@
+import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/query/react';
+import { getToken } from '../../auth/settings';
+
+
+export const ApplicationApi = createApi({
+  reducerPath: 'ApplicationApi',
+  tagTypes: ['Applications'],
+  baseQuery: fetchBaseQuery({
+    baseUrl: process.env.REACT_APP_BASE_URL,
+    prepareHeaders: async (headers) => {
+      const access_token = await getToken()
+      if (access_token) {
+        headers.set("Authorization", "Bearer " + access_token);
+      }
+    }
+  }),
+  endpoints: (builder) => ({
+    getPosts: builder.query({
+      query: (status = "APPROVED") => `/applications?status=${status}`,
+       providesTags: (result) =>
```

Comment:
As we have neither queries that provide tags for specific applications by ID nor mutations that invalidate them, I propose to keep it simple for now:
```
providesTags: ["Applications"]
invalidatesTags: ["Applications"]
```